src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -458,6 +458,13 @@ protected function instantiateObject(array &$data, string $class, array &$contex
 
         unset($context['has_constructor']);
 
+        if ($reflectionClass->isInstantiable() === false) {
+            throw NotNormalizableValueException::createForUnexpectedDataType(
```

Comment:
Wasn't sure if a new exception would be better or a different one may be better so let me know your thoughts

Reply:
This exception looks good to me

Reply:
Thanks 👍